app/helpers/menu_bar_helper.rb

```diff
@@ -17,6 +17,6 @@ def link_to_application(icon, app_name, minimal_permissions)
     app = Mumukit::Platform.application_for(app_name)
     url = app.organic_url(Organization.current.name)
 
-    link_to fixed_fa_icon(icon, text: t(app_name)), url
+    link_to fixed_fa_icon(icon, text: t(app_name)), url, {role:"menuitem", tabindex:"-1"}
```

Comment:
Don't use  `{` unless necessary. 

Also, prefer `'` over `"` for static string literals.